maintainers/scripts/gnome.sh

```diff
@@ -170,23 +170,23 @@ if [ "$action" == "update-all" ]; then
       echo "Skipping $project"
     else
       echo "= Updating $project to $majorVersion" >&2
-      update_project $project $majorVersion
+      update_project "$project" "$majorVersion"
       echo >&2
     fi
   done
 else
-  project="$2"
-  majorVersion="$3"
+  project=$2
```

Comment:
You meant everywhere instead of anywhere? It's your opinion that this is bad style. All you need to do is understand the rules. If the style is that correctness is the most important quality combined with a minimum number of characters under the condition that variables can contain spaces (the general rule is more complicated), then my style (which happens to coincide with the style of experts) is the one to follow. 

In the past few years some silly megacorporations have written style guides, which contradict this style in order to save a few dollars on staffing costs (because ignorant people are cheaper). In many cases the style guides have been written by people who themselves don't know what they are doing. I'd prefer it if we did not have people touching code who have to be explained this. 

To be honest, I think it's a bit silly to waste my time on even explaining this stuff. My version is more correct that the original version. Why don't you just merge the commit instead of starting a non-productive discussion?